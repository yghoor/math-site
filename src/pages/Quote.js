const Quote = () => (
  <div className="quote-page">
    <p>
      Mathematics is not about numbers, equations, computations, or algorithms: it is about
      understanding.
      <br/>
      <br/>
      - William Paul Thurston
    </p>
  </div>
);

export default Quote;