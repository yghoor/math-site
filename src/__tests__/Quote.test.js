/* eslint-disable no-unused-vars */
import renderer from 'react-test-renderer';

import Quote from '../pages/Quote.js';

test('renders "Quote" page', () => {
  const tree = renderer
    .create(<Quote />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});