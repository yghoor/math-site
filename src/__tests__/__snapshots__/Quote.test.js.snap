// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders "Quote" page 1`] = `
<div
  className="quote-page"
>
  <p>
    Mathematics is not about numbers, equations, computations, or algorithms: it is about understanding.
    <br />
    <br />
    - William Paul Thurston
  </p>
</div>
`;
